silq/pulses/pulse_sequences.py

```diff
@@ -1168,41 +1168,35 @@ def generate(self):
 
 
 class NMRCPMGPulseSequence(NMRPulseSequence):
-    """`PulseSequenceGenerator` for nuclear magnetic resonance (NMR).
+    """`PulseSequenceGenerator` for nuclear magnetic resonance CP sequences.
```

Comment:
I would prefer to say DD (dynamical decoupling) rather than CP/CPMG, but I defer to you on which it should be.

Reply:
I think that DD is a broader statement. For example, say you want to do CDD, then this parameter won't generate the pulse sequence you need.